conf/tests/test_wgs.config

```diff
@@ -7,11 +7,10 @@ process {
 }
 
 params {
-    submission_basepath = "grz-mini-test-data/submissions/wgs"
+    submission = "grz-mini-test-data/submissions/wgs"
     outdir = "${projectDir}/tests/results"
-    thresholds = "${projectDir}/assets/default_files/thresholds.json"
-    fasta = "grz-mini-test-data/references/GRCh37/mini.fa.gz"
-    target = "grz-mini-test-data/references/GRCh37/mini.genes.bed"
+    genomes.GRCh37.fasta = "grz-mini-test-data/references/GRCh37/mini.fa.gz"
```

Comment:
do you have a case with none of the reference is defined? (using the saved directory only)